vagga.yaml

```diff
@@ -0,0 +1,25 @@
+containers:
+    gdg_dev:
+      setup:
+      - !Ubuntu trusty
+      - !UbuntuUniverse
+      - !Install [software-properties-common, npm, curl, git, openssl, libssl-dev, libpq-dev, ca-certificates, build-essential, wget, python-pip, python-dev, mercurial]
```

Comment:
I think I can remove such pachages as mercurial, npm and wget, and mayby git, but the rest were added because they were needed to install main required packages. I'll try to get rid of them using renewed config.